dev/functional/tests/cypress/integration/02-institution/04-manage-location.spec.ts

```diff
@@ -105,7 +86,7 @@ describe("Manage Locations", () => {
         .phoneInputText()
         .clear()
         .type(data.phoneNumber);
-      institutionManageLocationObject.submitButton().should("be.enabled");
+      institutionManageLocationObject.submitButton().should("be.disabled");
```

Comment:
This type of UI behavior is suitable to be changed in the future, if changed the tests will be broken.
I would recommend avoiding going deeper into UI-specific behaviors unless we have a confirmation that it is final or close to final.